Dockerfile

```
@@ -46,31 +46,30 @@ Pin-Priority: 1000 \n\
  && apt-get update \
  && apt-get install -y --no-install-recommends firefox \
 # Install geckodriver, webdriver for firefox, needed for selenium
- && curl --proto "=https" -L https://github.com/mozilla/geckodriver/releases/download/v0.30.0/geckodriver-v0.30.0-linux64.tar.gz | tar xz -C /usr/local/bin \
+ && curl --proto "=https" -L "https://github.com/mozilla/geckodriver/releases/download/v0.30.0/geckodriver-v0.30.0-linux64.tar.gz" | tar xz -C /usr/local/bin \
 # Install health-checker tool that allows us to run commands when checking root endpoint to check if service is available
- && wget -q https://github.com/gruntwork-io/health-checker/releases/download/v0.0.8/health-checker_linux_amd64 -O /usr/local/bin/health-checker \
+ && curl --proto "=https" -L "https://github.com/gruntwork-io/health-checker/releases/download/v0.0.8/health-checker_linux_amd64" -o /usr/local/bin/health-checker \
  && chmod +x /usr/local/bin/health-checker \
 # Cleanup image and remove junk
  && rm -fr /var/lib/apt/lists/* \
 # Remove unused packages. Keep curl for health checking in docker-compose
- && apt-get purge -y ca-certificates wget
+ && apt-get purge -y ca-certificates
 
 # Create stored data dir inside image, in case it does not get mounted (such as when deploying on AWS)
 RUN mkdir /stored_data \
     && setfacl -R -d -m u:nonroot:rwx /stored_data \
     && setfacl -R -m u:nonroot:rwx /stored_data \
-    && mkdir /stored_data/geoserver \
-    && mkdir /stored_data/model_output
-
-USER nonroot
+    && mkdir /stored_data/geoserver
 
 # Copy python virtual environment from build layer
-COPY --chown=nonroot:nonroot --chmod=544 --from=build /venv /venv
+COPY --chmod=555 --from=build /venv /venv
+
```

As we discussed, let's add the `chown` command to be more explicit.
```suggestion
COPY --chown=root:root --chmod=555 --from=build /venv /venv
```